lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -119,6 +120,8 @@ static int nvme_queue_cmd(struct nvme_blockdev *bdev, struct spdk_nvme_qpair *qp
 			  struct nvme_blockio *bio,
 			  int direction, struct iovec *iov, int iovcnt, uint64_t nbytes,
 			  uint64_t offset);
+void
```

Comment:
Prototypes should not have a line break between the type and the function name.

Can you just reorder the definition of the timeout function so that it doesn't need a pre-declaration?